src/main/java/org/jabref/gui/collab/metedatachange/MetadataChangeDetailsView.java

```diff
@@ -0,0 +1,30 @@
+package org.jabref.gui.collab.metedatachange;
+
+import javafx.scene.control.Label;
+import javafx.scene.layout.VBox;
+
+import org.jabref.gui.collab.ExternalChangeDetailsView;
+import org.jabref.logic.l10n.Localization;
+import org.jabref.preferences.PreferencesService;
+
+public final class MetadataChangeDetailsView extends ExternalChangeDetailsView {
+
+    public MetadataChangeDetailsView(MetadataChange metadataChange, PreferencesService preferencesService) {
+        VBox container = new VBox(15);
+
+        Label header = new Label(Localization.lang("The following metadata changed:"));
```

Comment:
I would also make it parameterized here with %0

Reply:
The metadata changes*(or parameters)* will be displayed in separate labels.

Reply:
Ah okay!